### PR TITLE
Correction of links

### PR DESCRIPTION
**Pull request by CsatariGergely**
I'm working on a pr to build automatic link checking. During the testing of it I had to correct a couple of links...

### CODE REVIEW

**Comment by liamfallon** [comment]
Could we get one more approval on this before merging?

File: content/en/docs/guides/install-guides/install-on-byoc.md
Comment (efiacor) [inline content/en/docs/guides/install-guides/install-on-byoc.md:1]: Should we avoid having links in table items?

**Review by efiacor — COMMENTED** [review]
Minor comment

File: content/en/docs/guides/install-guides/install-on-byoc.md [hunk 1]
```diff
@@ -115,11 +115,11 @@ Flux or ArgoCD to apply packages to the clusters.
...
-| Provider                    | Notes                                                                                   |
-| --------------------------- | --------------------------------------------------------------------------------------- |
-| [Cluster API](capi.md)      | Kubernetes project cluster provisioner for a variety of cluster providers.              |
-| [KCC](kcc.md)               | Google's Kubernetes Config Connector for GKE clusters and other GCP resources.          |
-| [Crossplane](crossplane.md) | API composition framework with cluster and other infrastructure providers.              |
+| Provider    | Notes                                                                                   |
+| ----------- | --------------------------------------------------------------------------------------- |
+| Cluster API | Kubernetes project cluster provisioner for a variety of cluster providers.              |
+| KCC         | Google's Kubernetes Config Connector for GKE clusters and other GCP resources.          |
+| Crossplane  | API composition framework with cluster and other infrastructure providers.              |
```
Comment (efiacor) [inline content/en/docs/guides/install-guides/install-on-byoc.md:122]:
```suggestion
| Provider    | Notes                                                                                   |
| ----------- | --------------------------------------------------------------------------------------- |
| [Cluster API](https://cluster-api.sigs.k8s.io/) | Kubernetes project cluster provisioner for a variety of cluster providers.              |
| [KCC](https://cloud.google.com/config-connector/docs/overview)         | Google's Kubernetes Config Connector for GKE clusters and other GCP resources.          |
| [Crossplane](https://docs.crossplane.io/latest/getting-started/introduction/)  | API composition framework with cluster and other infrastructure providers.              |
```

File: content/en/docs/guides/user-guides/_index.md [hunk 2]
```diff
@@ -330,10 +330,10 @@ The diagram below depicts deployment at the high level.
 
 On the demo VM, the `test-infra/e2e/provision/hacks` directory contains some
 workaround scripts. Also, please see the Release Notes
-[known issues]({{< relref "../release-notes/r1#known-issues-and-workarounds">}}).
+[known issues]({{< relref "../../release-notes/r1#known-issues-and-workarounds">}}).
 
 ## Next Steps
 
-* Learn more about the Nephio [controllers]({{< relref "controllers.md">}})
+* Learn more about the Nephio controllers
```
Comment (liamfallon) [inline content/en/docs/guides/user-guides/_index.md:337]: Do we need to keep a link here?

Comment (CsatariGergely) [reply content/en/docs/guides/user-guides/_index.md:337]: Yes and no :)
Yes, in the final documentation we need a link there.
But I forgot tho move the `controllers.md` file and the link is broken now. 
I move back the file and restore the link in https://github.com/nephio-project/docs/pull/77

Comment (liamfallon) [reply content/en/docs/guides/user-guides/_index.md:337]: > But I forgot tho move the `controllers.md` file and the link is broken now.

OK, I'll check on #77 and see if we can get it merged.

Comment (liamfallon) [reply content/en/docs/guides/user-guides/_index.md:337]: > > But I forgot tho move the `controllers.md` file and the link is broken now.
> 
> OK, I'll check on #77 and see if we can get it merged.

#77 is merged now so maybe we can fix the link.

Comment (CsatariGergely) [reply content/en/docs/guides/user-guides/_index.md:337]: Thanks

Comment (CsatariGergely) [reply content/en/docs/guides/user-guides/_index.md]: I've rebased [60878b4](https://github.com/nephio-project/docs/pull/90/commits/60878b4a03d1bf220c5e19c9a2c9e888113c930f) and a couple other commits (plus fixed some new link errors :))

**Pull request merged**